Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM registry.centos.org/sclo/nodejs-4-centos7:latest
+
+# Install dependencies for mattermost-integration-github
+RUN git clone https://github.com/gorkem/penfold &&\
+    cd penfold && git checkout 82ac02228db306af6375ca2db08f0dbde5bc3f44 &&\
```

Comment:
Does this have to checkout the specific SHA? Can we build master. This particular SHA is already old now.

Reply:
removed building from specific commit part 
so now it will build the master branch